game/src/main/scala/io/aigar/game/EntityContainer.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace.Vector2
+
+
+trait EntityContainer {
+  def respawn(grid: Grid, players: List[Player], respawnRetryAttempts: Int): Option[Vector2] = {
```

Comment:
Could we rename this to something else? It is not really respawning anything, it is finding a respawn position.
Suggestion: `getRespawnPosition`